### DISCUSSION TITLE
article/notion-next-guide

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#57] [Announcements]
# article/notion-next-guide

介绍NotionNext的所有功能和常见问题说明

https://tangly1024.com/article/notion-next-guide

**Comment by AShanWang** [comment]
你好！
对于加密的文章，请问如何实现光标自动定位到密码框！

例如我在PC端打开加密的文章后，可以直接通过键盘输入密码；
而不是每次用鼠标点击密码框后才能输入密码!!!

**Reply by tangly1024** [reply]
目前不支持自动定位到密码框，但我可以加上这个功能

**Reply by AShanWang** [reply]
> 目前不支持自动定位到密码框，但我可以加上这个功能

好的，了解！
膜拜大佬🙏

**Reply by tangly1024** [reply]
新版已发布，支持自动定位到密码框。相关[Github代码](https://github.com/tangly1024/NotionNext/pull/1151) ，访问 [更新日志](https://tangly1024.com/article/notion-next-changelogs)

**Reply by AShanWang** [reply]
> 新版已发布，支持自动定位到密码框。相关[Github代码](https://github.com/tangly1024/NotionNext/pull/1151) ，访问 [更新日志](https://tangly1024.com/article/notion-next-changelogs)

哦！太棒了。👍

已更新体验，很丝滑。

**Comment by liguanghe** [comment]
你好，之前已经用github 和github pages 搭建了hexo主题的博客，数据和部署是在本地电脑上，现在思考直接在 notion.ai 上生成文章和上传到博客。在想这两种方式有哪些异同，是否可以同时使用呢？不需要导入和导出源文本，并且能同步更新是最好的。

**Reply by tangly1024** [reply]
我目前只专注在Notion中写作。也许您可以尝试《[基于 Notion 的笔记写作和博客分享自动化](https://hexo.kaedea.com/2021/05/20/devops/notion_to_markdown_file_automating_solution/)》这个方案，保持Notion中的博客与Hexo-Github同步。

**Reply by LetTTGACO** [reply]
如果是Notion + Hexo的云端写作本地部署的方案，可以考虑下[Elog](https://elog.1874.cool/)😝

**Comment by gaoran1209** [comment]
你好，请问我在 github 上替换站点 icon 成功，但实际网站上未生效。
请问该如何处理？
我的站点是：https://blog.gaoran.xyz/

**Comment by 369386973** [comment]
error - Conflicting paths returned from getStaticPaths, paths must be unique per page.
See more info here: https://nextjs.org/docs/messages/conflicting-ssg-paths
path: "/search" from page: "/[...slug]" conflicts with path: "/search" 
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Command "yarn run build" exited with 1
BUILD_UTILS_SPAWN_1: Command "yarn run build" exited with 1

在主题文件config_hexo.js和语言lang文件中提交代码部署总会出现这个错误

**Reply by 369386973** [reply]
现在就是notion的图片无法显示，视频可以正常显示

**Reply by tangly1024** [reply]
图片问题是bug，已提交了修复补丁，更新体验

**Comment by caijunqian** [comment]
请问有计划支持<b>多级分类</b>吗

**Comment by wzf19901221666** [comment]
请问一下 ，想在blog,config,js那边做修改，但是那边编辑选项是不能按的，还不能保存，让我加一个BRANCH  这个怎么操作呢

**Reply by tangly1024** [reply]
可能是未登录，或者因为你开的是别人的仓库所以没有编辑权限，[修改配置](https://github.com/wzf19901221666/NotionNext/edit/main/blog.config.js)

**Comment by PandaUMoon** [comment]
怎么设置文章侧面有目录导航

**Comment by x420520** [comment]
希望可以添加NotionNext= GitHub action的部署。尽可能少依赖第三方，这样可以减少出问题的可能性。

**Comment by Anson20198** [comment]
请问为什么浏览器站点图标在上传文件后没有更改

**Comment by leonwong282** [comment]
怎么 notion 修改后，网页没有修改
